### PR TITLE
Configuring logging from karaf distribution of Apache Brooklyn

### PR DESCRIPTION
**Pull request by bostko**

### CODE REVIEW

File: guide/dev/tips/logging.md [hunk 1]
```diff
@@ -11,6 +11,10 @@ with a default configuration which just works out of the box
 and bindings to the other common libraries (``java.util.logging``, ``log4j``, ...)
 if you prefer one of those.
 
+Modify `etc/org.ops4j.pax.logging.cfg`.
```
Comment (geomacy) [inline guide/dev/tips/logging.md]: Needs to be in a separate section for Karaf.  Karaf log control is not based on logback. And in any case with Karaf you would be better to explain about use of Karaf console to set log levels (use `log:set` command).  The only time you would need to modify the file by hand would be if for some reason it needed modified before the first start of Karaf.

**Comment by tbouron** [comment]
@bostko Can you update the PR as per as @geomacy's comment?

File: guide/dev/tips/logging.md (hunk 1 above)
Comment (bostko) [inline guide/dev/tips/logging.md]: @geomacy to what extend logging is configurable for OSGi distribution through config files in `brooklyn-logback-includes` module?

**Comment by bostko** [comment]
@tbouron I included @geomacy note.

**Pull request merged**